book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] In this method, the functionality to create a rental is already domiciled in the constructor of the `Rental` class. Simply ensure that your method accepts a `person` parameter, then initialize a new `Rental` class as shown below 👇 
#### DO THIS INSTEAD
```ruby
def add_rental(person, date)
  Rental.new(self, person, date)
end
```